mmcls/core/visualization/image.py

```diff
@@ -0,0 +1,108 @@
+import matplotlib.pyplot as plt
+import mmcv
+import numpy as np
+
+EPS = 1e-2
+
+
+def color_val_matplotlib(color):
+    """Convert various input in BGR order to normalized RGB matplotlib color
+    tuples,
+
+    Args:
+        color (:obj:`Color`/str/tuple/int/ndarray): Color inputs
+
+    Returns:
+        tuple[float]: A tuple of 3 normalized floats indicating RGB channels.
+    """
+    color = mmcv.color_val(color)
+    color = [color / 255 for color in color[::-1]]
+    return tuple(color)
+
+
+def imshow_cls_result(img,
+                      result,
```

Comment:
replace the "result" with "infos" or others, some text add to image may not be result.